src/pages/AppList/AppListComponent.tsx

```diff
@@ -143,6 +144,15 @@ class AppListComponent extends FilterComponent.Component<AppListComponentProps,
           ]}
         />
       </VirtualList>
+    ) : (
+      <EmptyState variant={EmptyStateVariant.full}>
```

Comment:
What do you think of moving this emptystate to a separate component? The behavior between other list pages are really similar.